todoapp/src/styles/styles.ts

```diff
@@ -0,0 +1,175 @@
+import { makeStyles} from '@mui/styles';
+import background from "../assets/backdrop.jpg";
+
+const useStyles = makeStyles({
+  root: {
+    "&&": {
```

Comment:
If this style file is generic for all then this name is ok otherwise its name should be like todoStyles.